src/Raven.Server/Monitoring/Snmp/SnmpOids.cs

```diff
@@ -413,11 +422,20 @@ private Databases()
             [Description("Remaining storage disk space in MB")]
             public const string StorageDiskRemainingSpace = "5.2.{0}.2.6";
             
-            [Description("IO read per seconds")]
-            public const string StorageDiskIoRead = "5.2.{0}.2.7";
+            [Description("IO read operation per seconds")]
+            public const string StorageDiskIoReadOperations = "5.2.{0}.2.7";
+            
+            [Description("IO write operation per seconds")]
+            public const string StorageDiskIoWriteOperations = "5.2.{0}.2.8";
+            
+            [Description("Read throughput in kilobyte pe seconds")]
+            public const string StorageDiskReadThroughput = "5.2.{0}.2.9";
+            
+            [Description("Write throughput in kilobyte pe seconds")]
+            public const string StorageDiskWriteThroughput = "5.2.{0}.2.10";
             
-            [Description("IO write per seconds")]
-            public const string StorageDiskIoWrite = "5.2.{0}.2.8";
+            [Description("Queue length")]
```

Comment:
I wonder if we should say "Disk queue length" instead?

Reply:
I wondered about that as well but then to be consistent I need to add "Disk" to all of them.
I decided to not add to any of them but if you want something else I don't mind